### ISSUE TITLE
Add more feed filtering options

### ISSUE THREAD

**Issue by 0xtrr** [hzrd149/nostrudel#225]
With the recent "replyguy" spam attack, it recognized that it would be great to be able to filter my own feed, other than just muting words. I have no idea how hard this is to implement without slowing things down or decrease overall UX but here is a couple of alternatives:

- OP pubkey
- Profile pic url (Amethyst has this feature, it's niche but maybe it could offer a good way to apply defence against a bunch of spam accounts)

**Comment by DireMunchkin** [comment]
Yes this is really aggravating on noStrudel as of late - Filtering replies with a Web of Trust score filter would also be a possible solution.

**Comment by 0xtrr** [comment]
Some other filters that could help users remove spam:

1. Filter out posts from npubs where the NIP-05 check fails.
2. Filter out posts from npubs using a certain NIP-05 domain.
3. Filter out posts from npubs that haven't set up a NIP-05

Not sure how smart/good number 3 is but maybe it makes sense in some scenarios?

**Comment by Giszmo** [comment]
Instead of filtering by default or asking the user to enable filtering, please sort by default by WoT score. My direct follows first. Follows of follows next. Others later but hidden by default. Click to show.
